test/cypress/plugins/index.js

```diff
@@ -14,193 +22,41 @@ const cookieMonster = require('../integration/utils/cookie-monster')
-  )
-  config.env.encryptedSessionRequestToGoLiveStageTermsAgreedEpdqCookie = encryptedSessionRequestToGoLiveStageTermsAgreedEpdqCookie
-  config.env.encryptedGatewayAccountRequestToGoLiveStageTermsAgreedEpdqCookie = encryptedGatewayAccountRequestToGoLiveStageTermsAgreedEpdqCookie
-
-  // REQUEST_TO_GO_LIVE_STAGE_DENIED user
-  const selfServiceRequestToGoLiveStageDeniedUser = selfServiceUserConfig.config.users.find(element => element.cypressTestingCategory === 'REQUEST_TO_GO_LIVE_STAGE_DENIED')
-  const encryptedSessionRequestToGoLiveStageDeniedCookie = generateEncryptedSessionCookie(
-    config.env.TEST_SESSION_ENCRYPTION_KEY,
-    selfServiceRequestToGoLiveStageDeniedUser.external_id
-  )
-  const encryptedGatewayAccountRequestToGoLiveStageDeniedCookie = generateEncryptedGatewayAccountCookie(
-    config.env.TEST_SESSION_ENCRYPTION_KEY,
-    selfServiceRequestToGoLiveStageDeniedUser.gateway_accounts.find(fil => fil.isPrimary === 'true').id
-  )
-  config.env.encryptedSessionRequestToGoLiveStageDeniedCookie = encryptedSessionRequestToGoLiveStageDeniedCookie
-  config.env.encryptedGatewayAccountRequestToGoLiveStageDeniedCookie = encryptedGatewayAccountRequestToGoLiveStageDeniedCookie
-
-  // REQUEST_TO_GO_LIVE_STAGE_LIVE user
-  const selfServiceRequestToGoLiveStageLiveUser = selfServiceUserConfig.config.users.find(element => element.cypressTestingCategory === 'REQUEST_TO_GO_LIVE_STAGE_LIVE')
-  const encryptedSessionRequestToGoLiveStageLiveCookie = generateEncryptedSessionCookie(
-    config.env.TEST_SESSION_ENCRYPTION_KEY,
-    selfServiceRequestToGoLiveStageLiveUser.external_id
-  )
-  const encryptedGatewayAccountRequestToGoLiveStageLiveCookie = generateEncryptedGatewayAccountCookie(
-    config.env.TEST_SESSION_ENCRYPTION_KEY,
-    selfServiceRequestToGoLiveStageLiveUser.gateway_accounts.find(fil => fil.isPrimary === 'true').id
-  )
-  config.env.encryptedSessionRequestToGoLiveStageLiveCookie = encryptedSessionRequestToGoLiveStageLiveCookie
-  config.env.encryptedGatewayAccountRequestToGoLiveStageLiveCookie = encryptedGatewayAccountRequestToGoLiveStageLiveCookie
+  on('task', {
+    getCookies (opts) {
+      const encryptedSessionCookie = generateEncryptedSessionCookie(
+        config.env.TEST_SESSION_ENCRYPTION_KEY,
+        opts.user_external_id
+      )
+      const encryptedGatewayAccountCookie = generateEncryptedGatewayAccountCookie(
+        config.env.TEST_SESSION_ENCRYPTION_KEY,
+        opts.gateway_account_id
+      )
+
+      return { encryptedSessionCookie, encryptedGatewayAccountCookie }
+    },
+    setupStubs (stubSpecs) {
+      let stubsArray = []
+      stubSpecs.forEach((spec) => {
+        stubsArray = lodash.concat(stubsArray, stubs[spec.name](spec.opts))
+      })
+
+      return baseClient.post({
+        url: 'http://localhost:2525/imposters',
+        json: true,
+        body: {
+          port: 8000,
+          protocol: 'http',
+          stubs: stubsArray
+        }
+      })
```

Comment:
we post to Mountebank to setup the stubs. I'll clean this up a bit to put the address in a config file at some point